### PR TITLE
Add documentation about organisation

### PR DESCRIPTION
**Pull request by sjib**
- Should help with https://github.com/teksi/wastewater/issues/57
- Should help prepare use of VSA-DSS 2015 import/export

### CODE REVIEW

File: en/user-guide/layerexplanations/layerexplanation.rst [hunk 1]
```diff
@@ -57,7 +57,19 @@ The channel-class has no geometry and is therefore mostly changed in the vw_qgep
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-todo: This table is today at a strange place (it is not only related to wastewater structures) and has to be revised.
+od_organisation contains the organisation that you can select in attributes like fk_dataowner, fk_operator, fk_provider, fk_owner, etc.
+
+ .. figure:: images/od_organisation.jpg
+
+This table is today a little bit hidden in the wastewater_structures group (but it is not only related to wastewater structures).
+
+If you want to export data to the VSA-DSS 2015 model, you have to manually add the relation to the respecitve subclasses of organisation by adding the respective obj_id's in the subclass tables, e.g. municipality, else the export will give an error message. For export to SIA405_Abwasser and VSA-KEK tthis is not needed.
```
[the remark this answers is not in the document]
Comment (sjib) [reply en/user-guide/layerexplanations/layerexplanation.rst]: Thanks - I have updated the text accoringly.

**Comment by sjib** [comment]
@urskaufmann Ok like this?

**Review by urskaufmann — APPROVED** [review]
Thanks

File: en/user-guide/layerexplanations/layerexplanation.rst [hunk 2]
```diff
@@ -57,7 +57,19 @@ The channel-class has no geometry and is therefore mostly changed in the vw_qgep
...
+
+ .. figure:: od_organisation_postgres.jpg
```
[the remark this answers is not in the document]
Comment (sjib) [reply en/user-guide/layerexplanations/layerexplanation.rst:68]: Thanks - I will fix in a new pull request

**Pull request merged**